modules/vault/options.go

```diff
@@ -0,0 +1,82 @@
+package vault
+
+import "fmt"
+
+// Option is a function type for modifying a Vault configuration
+type Option func(*Config)
+
+// Config is a struct that contains various configuration options for the Vault
+type Config struct {
+	imageName    string              // The name of the Docker image to use for the Vault
+	token        string              // The root token to use for the Vault
+	port         int                 // The port number to use for the Vault
+	secrets      map[string][]string // A map of secret paths to their respective secret values
+	initCommands []string            // A list of commands to execute on Vault initialization
+	logLevel     LogLevel            // The level of logging to use for the Vault
```

Comment:
This is a env var which can be passed by override the `ContainerRequest`. I know this is handy but in previous conversation we have discussed this. At least there should be some logic around the env var such as in databases in order to take this into account. In Java, should be deprecated.